test-suite/Makefile

```diff
@@ -520,14 +519,6 @@ approve-coqdoc: coqdoc
 	  echo "Updated $$f!"; \
 	fi; done)
 
-# the expected output for the MExtraction test is
-# /plugins/micromega/micromega.ml except with additional newline
-output/MExtraction.out: ../plugins/micromega/micromega.ml
```

Comment:
Is this test just removed in the PR as it stands now?

IIUC, currently micromega.ml is generated by extraction of stdlib MExtraction.v. IMO that's a neat use of Coq to build Coq, and it'll be a shame if we have to let it go. What is the new plan for maintaining that ML code in the new dualrepo world?

Reply:
As can be seen in the diff, the test is not removed. It is only moved to the test-suite of the stdlib.
I added comments in both files to clarify the situation.

Reply:
And more generally about the micromega plugin, my hope is that in the mid/long term we manage to clarify the interface of the plugin. I'd like to work on this in the future, but that's definitely not a short term endeavour.